src/actions/compass.ts

```diff
@@ -0,0 +1,45 @@
+import {SpecQueryModelGroup} from 'compassql/build/src/model';
+import {Query} from 'compassql/build/src/query/query';
+import {Schema} from 'compassql/build/src/schema';
+import {Dispatch} from 'redux';
+import {ThunkAction} from 'redux-thunk';
+import {ReduxAction} from './redux-action';
+
+import {fetchCompassRecommends} from '../api/api';
+import {State} from '../models/index';
+import {getQuery, getSchema} from '../selectors';
+import {Action} from './index';
+
+export type CompassAction = CompassRecommendsRequest | CompassRecommendsRecieve;
```

Comment:
Good question.  I think ResultRequest / ResultReceive sound good to me.  

ResultAction sounds a bit weird, but I guess that's better than having `QueryAction = ResultRequest | ResultReceive`.